### PR TITLE
mkdocs: initial html documentation

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

**Comment by m4ra** [comment]
@goapunk we need to get the mkdocs.yml in the main branch and then I can generate the dedicated github branch for deploying the documentation website on github pages

File: requirements/dev.txt [hunk 1]
```diff
@@ -6,6 +6,8 @@ Faker==18.10.1
 flake8==6.0.0
 freezegun==1.2.2
 isort==5.12.0
+mkdocs==1.6.0
+mkdocstrings==0.24.3
```
Comment (goapunk) [inline requirements/dev.txt:10]: this needs to be `mkdocsstrings[python]`, otherwise I get an error

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Welcome to MkDocs
+
+For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+
+## Commands
+
+* `mkdocs new [dir-name]` - Create a new project.
+* `mkdocs serve` - Start the live-reloading docs server.
```
Comment (goapunk) [inline docs/index.md:8]: should we add this one to the Makefile?

**Review by goapunk — COMMENTED** [review]
some more comments

File: docs/index.md (hunk 2 above)
Comment (m4ra) [reply docs/index.md:8]: They don't require the make command, so I would leave these here for now.

**Comment by m4ra** [comment]
@goapunk I addressed your comments, added info from the README in the index page, and change the theme.

**Review by goapunk — APPROVED** [review]
Nice! LGTM

**Pull request merged**